proposal/09510/README.rst

```diff
@@ -0,0 +1,64 @@
+========================
+Encryption Acceleration 
+========================
+
+
+Basic Information
+========================
+Code repository: https://github.com/09510/Encryption-Acceleration
+
+Problem to solve
+=======================
+Message encryption is an important technology to protect information security. 
+However, if you want to ensure the encryption strength, the encryption/decryption takes a lot of time.
+Encryption Acceleration is a tool to accelerate encryption and decryption by GPU. 
+
+
+
+Perspective users 
+=========================
+Anyone who wants to use GPU to accelerate encryption and decryption.
```

Comment:
If you only address those who use GPU like what you put here, the goal greatly misaligns with the purpose of what I want you to practice with the term project.